lib/std/mem.zig

```diff
@@ -258,6 +258,38 @@ pub fn zeroes(comptime T: type) T {
     }
 }
 
+/// Compares two slices in constant time (for a given length) and returns whether they are equal.
```

Comment:
Two slices? Did you mean arrays?
Slices are easily supported by slicing them with a fixed start/end as that gives you a `*[N]T`

Reply:
The comment has been fixed, thank you!